packages/blade/src/components/Badge/Badge.stories.tsx

```diff
@@ -50,7 +51,21 @@ const Page = (): ReactElement => {
       <br />
       <br />
       <Title>Usage</Title>
-      <Highlight language="tsx">{`import { Badge } from '@razorpay/blade/components'; \nimport type { BadgeProps } from '@razorpay/blade/components';`}</Highlight>
+      <Sandbox>
```

Comment:
we need to add this change rest of the stories too right? are you gonna add it in this PR only?

Reply:
Nope will add it in followup PRs after Figma's PR. Both sandbox and figma requires changes in other stories so will do it together.